content/handbook/product/personas/_index.md

```
@@ -0,0 +1,687 @@
+stale: false
+---
+
+## ロールとペルソナの違い
+
+ペルソナは GitLab にとっての理想的なターゲットを表します。メッセージングやマーケティングの提供方法を定義するのに役立ちます。彼らはターゲットとする理論上の人物です。彼らの関心事や情報の収集先を定義することで、この理想的なターゲットに焦点を当てて、マーケティング費用とセールスの取り組みを最適に投じることができます。
+
+ロールは個別の役職です。これらは、セールスの過程で実際に出会う人々です。アカウントの中で特定のロールを持つ連絡先を見つけることになります。IT における各ロールが直面する課題と、彼らが最も関心を持っているものを理解することは、適切な人に適切な価値提案を届けるのに役立ちます。
+
+## ペルソナ
+
+ペルソナは、コミュニケーションし設計するターゲットとなる理想像について語る一般化された方法です。メッセージングやマーケティング提供だけでなく、製品の定義にも役立ちます。ペルソナを念頭に置くことで、適切な言葉を使い、彼らの特定の問題や苦痛点に対処するための最善の決定を下すことができます。GitLab には [購入者](#buyer-personas) と [ユーザー](#user-personas) の両方のペルソナタイプがあります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)